src/ray/raylet/node_manager.cc

```diff
@@ -765,6 +765,9 @@ void NodeManager::ProcessClientMessage(
   case protocol::MessageType::GetTask: {
     ProcessGetTaskMessage(client);
   } break;
+  case protocol::MessageType::TaskDone: {
+    ProcessGetTaskMessage(client);
```

Comment:
check `UsePush == true` here? so is `GetTask`.

Reply:
ok